app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
```

Comment:
Well given the discussions and the identified features in this [sketch](https://trello-attachments.s3.amazonaws.com/58386a14118e985eafa6e7e9/58b6ba1c1f5400325dcded4d/486a4666499d4e795a2851ac787eba9d/IMG_0584.JPG) from this [Trello card](https://trello.com/c/oVP2S6GC/172-5-filter-department).. The term dashboard appeared rational in this case.. I have also covered that in my commit notes f75e7794acea5eb5e0a29d304d1cae7c19fde18d.. I quote 

> This is presented on the root page of what is understood to be the
beginnings of a dashboard page. Minor change in the root route from
organisation to dashboard.